omnibus/config/software/postgresql.rb

```diff
@@ -0,0 +1,62 @@
+#
```

Comment:
At this point it looks like we might bre able to use the omnibus-software defition of postgres? Or asre we still stuck because of the version-specific prefix?
I'm wonder if we should just port that change over to the official definition?  Or make an alternative official definition (postgresql-versioned.rb?) that mirrors the pg one except for install path?